fdbserver/storageserver.actor.cpp

```diff
@@ -5994,7 +5989,10 @@ ACTOR Future<Void> update(std::shared_ptr<ptxn::StorageServer> storageServerCont
 		// Need a local copy of error as there is a wait in the catch clause
 		state Error error_ = error;
 		if (error.code() != error_code_worker_removed && error.code() != error_code_please_reboot) {
-			TraceEvent(SevError, "SSUpdateError", storageServerContext->thisServerID).error(error).backtrace();
+			TraceEvent(
+			    g_network->isSimulated() ? SevWarn : SevWarnAlways, "SSUpdateError", storageServerContext->thisServerID)
```

Comment:
Why this is no longer a `SevError` any more?

Reply:
This happens in PTXN more often, and is legit. If the initial recovery fails, a new round of SS recruitment happens, and the seed SS may be assigned different team. The old team will through `team not exist` error, which is normal.